quickshow/src/cellrenderer.cpp

```diff
@@ -180,8 +183,8 @@ ResultsCellRenderer::ResultsCellRenderer(const QSize &size, QWidget *widget)
 	m_cellAttributes.resize(ColumnCount);
 	m_cellAttributes[Position] = CellAttribute{4 * m_scaledLetterWidth, Qt::AlignRight};
 	m_cellAttributes[Name] = CellAttribute{0};
-	m_cellAttributes[Registration] = CellAttribute{7 * m_scaledLetterWidth};
-	m_cellAttributes[Time] = CellAttribute{7 * m_scaledLetterWidth, Qt::AlignRight};
+	m_cellAttributes[Registration] = CellAttribute{5 * m_scaledLetterWidth};
+	m_cellAttributes[Time] = CellAttribute{4 * m_scaledLetterWidth, Qt::AlignRight};
```

Comment:
What if we start to support msecs?

Reply:
There is plenty place for 0.1 and 0.01 of seconds, but OK, I'll return it back.